velox/type/TimestampConversion.cpp

```diff
@@ -144,6 +145,53 @@ bool isValidWeekDate(int32_t weekYear, int32_t weekOfYear, int32_t dayOfWeek) {
   return true;
 }
 
+bool isValidWeekOfMonthDate(
+    int32_t year,
+    int32_t month,
+    int32_t weekOfMonth,
+    int32_t dayOfWeek) {
+  if (year < 1 || year > kMaxYear) {
```

Comment:
Is it indeed the case that we do not allow specifying dates before 1970?

Reply:
In this validation, dates before **1** is not allowed, not 1970.